R/utils.R

```diff
@@ -661,9 +657,22 @@ greta_col <- function (which = c('main',
                                  'dark',
                                  'light',
                                  'lighter',
-                                 'super_light')) {
+                                 'super_light'),
+                       color = NULL) {
```

Comment:
could you please change to an explicit default here, rather than changing the NULL below:
```r
colour = "#996bc7"
```